### PR TITLE
Expanding the SSO FAQ

### PR DESCRIPTION
**Pull request by karensawrey**
As a follow-up for a ticket that required info that was known, but wasn't reflected in the docs. Fixing it with this PR.

### CODE REVIEW

File: pages/integrations/sso.md [hunk 1]
```diff
@@ -151,3 +151,10 @@ The second is that the account already exists in Buildkite. If you have access t
 ### Why do I get the error "we couldn't find an account with that email address" when logging in?
 
 This is likely caused by trying to log in from the wrong place. You need to log in from https://buildkite.com/sso and follow the link from the email you receive. If the issue persists, please reach out to support@buildkite.com for assistance.
+
+### Will setting the session duration affect all current sessions or only the new sessions?
+When you [update the session duration](https://buildkite.com/docs/apis/graphql/cookbooks/organizations#update-the-default-sso-provider-session-duration), it affects both the new and the old SSO sessions.
```
Comment (mbelton-buildkite) [inline pages/integrations/sso.md:156]: Formatting, link syntax, and an optional nitpick for brevity (removing "the"):

```suggestion
### Will setting the session duration affect all current sessions or only the new sessions?

When you [update the session duration](/docs/apis/graphql/cookbooks/organizations#update-the-default-sso-provider-session-duration), it affects both new and old SSO sessions.
```

File: pages/integrations/sso.md [hunk 2]
```diff
@@ -151,3 +151,10 @@ The second is that the account already exists in Buildkite. If you have access t
...
+
+### Which even is considered to be the start of an SSO session?
+
+The start of an SSO session is individually measured per user from the moment that user signs in via SSO.
```
Comment (mbelton-buildkite) [inline pages/integrations/sso.md:160]: I found this a bit confusing but might have simplified too much:

```suggestion
### When is an SSO session considered to start?

An SSO session starts for a user from the moment they sign in using SSO.
```

**Comment by karensawrey** [comment]
@MelissaKaulfuss - another one for you to approve. Many thanks in advance!

**Review by MelissaKaulfuss — APPROVED** [review]
✅ All good!

**Comment by MelissaKaulfuss** [comment]
@dannymidnight can you please deploy

**Pull request merged**